.github/workflows/ci.yml

```diff
@@ -68,8 +68,6 @@ jobs:
       - name: Install dependencies
```

Comment:
What do you prefer? Changing the name of this step to "Upgrade pip and setuptools" or moving it to line 79?

Reply:
We cannot move it, because the `pip install -e .` part is done for Linux and Mac together and for Windows specifically.
But changing the name is fine for me.

Reply:
Ok! I'm changing it.